### PR TITLE
SONARJAVA-4845 S6914: Use Fused Location to optimize battery power

### PR DESCRIPTION
**Pull request by kaufco**

### CODE REVIEW

File: sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json [hunk 1]
```diff
@@ -501,6 +501,7 @@
     "S6863",
     "S6889",
     "S6905",
-    "S6909"
+    "S6909",
+    "S6914"
```
Comment (ADarko22) [inline sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json:505]: Should we keep this rule in the Sonar Way? Let's double-check with @johann-beleites-sonarsource.
I think this is a valid rule, although forces to use Google API which may not be available on all devices.

**Review by ADarko22 — CHANGES_REQUESTED** [review]
Good job! Left a small comment to cover invocations in lambdas

File: sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json (hunk 1 above)
Comment (kaufco) [reply sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json:505]: I agree: should not be in Sonar Way profile.

**Review by ADarko22 — APPROVED** [review]
LGTM! 

Remember to update the RSPEC and remove the Sonar Way profile from there, otherwise the rule will be added to the profile when executing the `rule-api update`

**Pull request merged**